lib/msal-common/src/request/AuthorizationCodeUrlRequest.ts

```diff
@@ -77,8 +91,8 @@ export type AuthorizationCodeUrlRequest = {
     claims?: string;
 
     /**
-     *  A value included in the request that is also returned in the token response. A randomly
-     *  generated unique value is typically used for preventing cross site request forgery attacks.
+     *  A value included in the request that is returned in the id token. A randomly
+     *  generated unique value is typically used to mitigate replay attacks.
```

Comment:
General q. For type docs, shouldn't all comments be added before the type definition and not inside? I am not 100% sure, cc @pkanher617 since you deployed these for `msal-browser` can you please weigh in here?